installation-application/media_player.py

```diff
@@ -138,4 +137,4 @@ def terminate(self):
     VIDS_PATH = "./data/vids/"
     mp = MediaPlayer(GIFS_PATH, VIDS_PATH)
     mp.start_on_new_process()
-    mp.queue_media("vid-10")
\ No newline at end of file
+    mp.queue_media("vid-10")
```

Comment:
is this meant to be there?

Reply:
It's a half hearted test, when you run that file alone. Can be removed if you want

Reply:
ok bbz, gotta fix some other stuff then will merge <3